specification/src/main/asciidoc/managedbeans/Introduction.adoc

```diff
@@ -0,0 +1,29 @@
+== Introduction
+
+This specification defines Managed Beans for the Jakarta(TM) EE Platform.
+
+=== What are Managed Beans?
```

Comment:
This looks like a copy and paste from the JCP Managed Beans spec, which we haven't been
given permission to use.  I think all the following content needs to be removed.

Reply:
Yes, it is a copy-and-paste...  I really didn't think that this front matter would be a problem.  It's what our scope statement is based off of...  But, we think it will be a problem, then I can skinny it down to just the Scope statement.

Reply:
It's obviously copyright infringement...

Reply:
Okay, will remove...